modules/library/main/src/main/java/org/geotools/geometry/jts/GeometryClipper.java

```diff
@@ -163,6 +163,10 @@ public Geometry clip(Geometry g, boolean ensureValid) {
         if (g instanceof LineString) {
             return clipLineString((LineString) g);
         } else if (g instanceof Polygon) {
+            Polygon polygon = (Polygon) g;
```

Comment:
Is this used anywhere?

Reply:
To be removed, it was weekend code while I was trying to improve the MVT generation perf that got left on disk